site/src/pages/404.jsx

```diff
@@ -1,14 +0,0 @@
-import React from 'react'
```

Comment:
Does Astro add its own default 404 page ?

Reply:
Astro no longer supports JSX components. And I checked and netlify provided its own 404 so I don't believe this one is ever seen